src/naarad/utils.py

```diff
@@ -675,6 +675,7 @@ def get_standardized_timestamp(timestamp, ts_format):
     else:
       dt_obj = datetime.datetime.strptime(timestamp, ts_format)
       ts = calendar.timegm(dt_obj.utctimetuple())*1000 + dt_obj.microsecond/1000
+    ts = timestamp.replace(' ','_')
```

Comment:
Actually this is not what we want. This bypasses the conversion of a string timestamp to epoch milliseconds. We need the epoch milliseconds for comparison purposes otherwise we are comparing timestamp strings which won't be the correct comparison for comparing against `ts_start` and `ts_end` configurations.